app/models/institution_program.rb

```diff
@@ -43,14 +43,12 @@ def va_bah
   # Given a search term representing a partial school name, returns all
   # programs starting with the search term.
   #
-  def self.autocomplete(search_term, limit = 6)
-    joins(:institution).select('institution_programs.id, institutions.facility_code as value, description as label')
-                       .where(
-                         'lower(description) LIKE (?)',
-                         "#{search_term}%"
-                       )
-                       .group('institution_programs.id, institutions.facility_code, description')
-                       .limit(limit)
+  def self.autocomplete(search_term, version, limit = 6)
+    select(:id, 'institutions.facility_code as value', 'description as label')
+      .joins(institution: :version)
+      .where(institutions: { version: version })
+      .where('lower(description) LIKE (?)', "#{search_term}%")
+      .limit(limit)
```

Comment:
Now that we have the associations set up, we can use more ActiveQuery and less raw SQL